splunk/packages/edit-table/src/components/KVStoreUploader.jsx

```diff
@@ -114,9 +159,41 @@ export default function KVStoreUploader({
         }
     };
 
+    const runIncrementalUpload = async () => {
+        const data = await getFormattedData();
+        if (!data) return;
+
+        const incrementalUploadData = await getIncrementalUploadData(data);
+        const { dataForUpdate, dataForInsert } = incrementalUploadData;
+
+        try {
+            const options = { splunkApp, kvStore, backupErrorMsg, restoreErrorMsg };
+            const callback = async () => {
+                await batchUpsertCollectionEntries(
```

Comment:
how about calling `getIncrementalUploadData` inside of `batchUpsertCollectionEntries` and just passing `data`?

Reply:
the problem that I see here are the `*ErrorMsg` - how about we move them to data.js too?

Reply:
will be fixed in https://github.com/marcusschiesser/kveditor/issues/28